administration/scheduling-and-retries.md

```diff
@@ -10,6 +10,41 @@ Once an output plugin gets called to flush some data, after processing that data
 
 If the return status was **OK**, it means it was successfully able to process and flush the data, if it returned an **Error** status, means that an unrecoverable error happened and the engine should not try to flush that data again. If a **Retry** was requested, the _Engine_ will ask the _Scheduler_ to retry to flush that data, the Scheduler will decide how many seconds to wait before that happens.
 
+## Configuring Wait Time for Retry
+
+The Scheduler provides two configuration options called **scheduler.cap** and **scheduler.base** which can be set on service section. 
+
+| Key | Description | Default Value | 
+| -- | ------------| --------------| 
+| scheduler.cap | Set a maximum retry time in second. The property is supported from v1.8.7. | 2000 | 
+| scheduler.base | Set a base of exponential backoff. The property is supported from v1.8.7. | 5 |
+
+These two configuration determine the waiting time before retry happen. Fluent Bit use exponential-backoff-and-jitter algorithm to determine the waiting time. These configuration means when the Nth retry needed, scheduler will generate a random number X between `(min(base * (Nth power of 2), cap), cap)` as waiting time for this retry.Basically, the **scheduler.base** decide how fast retry could happen, the **scheduler.cap** decide what's the longest waiting time retry could be. 
```

Comment:
Can you give an example calculation to help folks understand? Also I think the formula on its own line is best.

Also I actually do not know what this formula means:

> scheduler will generate a random number X between `(min(base * (Nth power of 2), cap), cap)`

What are the upper and lower bounds for the random number generation? Is the `cap` the upper bound? I think we need a more clear way to write this.

Reply:
The example below does help

Reply:
Separate the upper and lower bound same as you do in the example, it makes it clearer

Reply:
Will separate the formula into independent line and add an example following it. 
Basically, lower bound would be the base, and upper bound will keep increasing with retry time goes until reach to the cap.